src/Core/Exception/HttpErrorCodeInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @since         5.2.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Core\Exception;
+
+use Throwable;
+
+interface HttpErrorCodeInterface extends Throwable
```

Comment:
Ideally this should have been in the `Http` package but couple of exceptions of the `Datasource` package are also mapped to 404 code :/

Reply:
I think this is ok. Hard to create a generic map with one way dependencies with out split packages.

Reply:
Do we get much from extending `Throwable`?

Reply:
Not really, I did it as this interface is only meant to be used by exception classes.